taui/src/components/neighborhood-details.js

```diff
@@ -53,86 +57,136 @@ export default class NeighborhoodDetails extends PureComponent<Props> {
     const bestJourney = neighborhood.segments && neighborhood.segments.length
       ? neighborhood.segments[0] : null
 
+    // Overall score is a derived value and not a neighborhood property (so not in `labels`).
+    const overallScore = neighborhood.score
+      ? neighborhood.score.toLocaleString('en-US', {style: 'percent'})
+      : message('UnknownValue')
+
     return (
-      <div className='Card'>
-        <div className='CardTitle'>
-          <Icon type={isFavorite ? 'star' : 'star-o'}
+      <div className='neighborhood-details'>
+        <header className='neighborhood-details__header'>
+          <Icon
+            className='neighborhood-details__star'
+            type={isFavorite ? 'star' : 'star-o'}
             onClick={(e) => setFavorite(id, userProfile, changeUserProfile)}
-            style={{cursor: 'pointer'}} />
-          <span>{town} - {id}</span>
+          />
+          {town} &ndash; {id}
+          <Icon className='neighborhood-details__marker' type='map-marker' />
+        </header>
+        <div className='neighborhood-details__trip'>
+          {Math.round(time)}&nbsp;
```

Comment:
Isn't `time` an integer?

Reply:
For some reason when user has a car, it's a float.

Reply:
Ah, okay. I'll go fix that in another PR later.